libmscore/types.h

```diff
@@ -258,7 +258,7 @@ enum class GlissandoType {
 //---------------------------------------------------------
 
 enum class GlissandoStyle {
-      CHROMATIC, WHITE_KEYS, BLACK_KEYS, DIATONIC
+      CHROMATIC, WHITE_KEYS, BLACK_KEYS, DIATONIC, CONTINUOUS
```

Comment:
For consistency, it'd be wise to use PORTAMENTO here as well See also the linked issue where this is (still being) discussed.